src/bacnet/bacenum.h

```diff
@@ -1619,11 +1619,14 @@ typedef enum BACnet_Services_Supported {
     SERVICE_SUPPORTED_UTC_TIME_SYNCHRONIZATION = 36,
     SERVICE_SUPPORTED_WHO_HAS = 33,
     SERVICE_SUPPORTED_WHO_IS = 34,
-    SERVICE_SUPPORTED_UNCONFIRMED_AUDIT_NOTIFICATION = 46,
+    SERVICE_SUPPORTED_UNCONFIRMED_AUDIT_NOTIFICATION = 46
     /* Other services to be added as they are defined. */
     /* All values in this production are reserved */
     /* for definition by ASHRAE. */
-    BACNET_SERVICES_SUPPORTED_MAX = 47
+
```

Comment:
The extra comment should reside in a pull request discussion, but not in the final code.  Please remove the comment (and the line of code that is being removed).